src/Raven.Client/Documents/Operations/Revisions/RevisionsBinConfiguration.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Raven.Client.Documents.Conventions;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.Revisions
+{
+    public sealed class RevisionsBinConfiguration : IDynamicJson
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether the revisions bin cleaner is disabled.
+        /// </summary>
+        /// <value>
+        /// <c>true</c> if the cleaner is disabled; otherwise, <c>false</c>.
+        /// </value>
+        public bool Disabled { get; set; }
+
+        /// <summary>
+        /// Gets or sets the minimum age of revisions-bin entries (deleted docs with revisions) to keep in the database.
+        /// The revisions-bin cleaner deletes the entries that are older than that.
+        /// </summary>
+        /// <value>
+        /// The minimum <see cref="TimeSpan"/> that revisions-bin entries (deleted docs with revisions) must be kept before being eligible for deletion.
+        /// A null value means no age restriction is applied.
+        /// </value>
+        public TimeSpan? MinimumEntriesAgeToKeep { get; set; }
```

Comment:
can we switch to int? TimeSpan is problematic for non-csharp languages

rename maybe to `long? MinimumEntriesAgeToKeepInMin` ? what unit would be good? InSec, InMin, InHrs, InDays?

Reply:
The name was inspired by this.. I think it should be consistent. what do you think?
![image](https://github.com/user-attachments/assets/a93d1077-5254-4ac0-92de-72255689d83e)

Reply:
Done